telegram/src/test/kotlin/com/github/kotlintelegrambot/network/apiclient/SendMediaGroupIT.kt

```diff
@@ -41,7 +41,7 @@ class SendMediaGroupIT : ApiClientIT() {
         val expectedRequestBody = String.format(
             getFileAsStringFromResources<SendMediaGroupIT>("sendMediaGroupRequestBody1.txt"),
             multipartBoundary
-        )
+        ).trimIndent()
```

Comment:
Why is this change needed?

Reply:
On windows machine we receive content with indent in the beginning, In real-life application they won't matter. 
![image](https://user-images.githubusercontent.com/32519327/109426240-a103ce00-79fd-11eb-9c11-2ee4b151e09f.png)
If there is more to dig, please point where to)

Reply:
Oh gotcha, good to know!